src/library/LibraryWindow.vala

```diff
@@ -169,7 +169,6 @@ public class LibraryWindow : AppWindow {
         // setup search bar and add its accelerators to the window
         search_entry = new SearchFilterEntry ();
         search_entry.valign = Gtk.Align.CENTER;
-        search_entry.activate.connect (() => {get_current_page ().grab_focus (); });
 
```

Comment:
You should not remove this - it has accessibility implications (cannot transfer keyboard focus to view by pressing Enter) and is not the root cause of the problem.

Reply:
You can always move focus with the `tab` key. I don't think we have other apps where we use `enter` as the pattern for moving focus